diff --git a/debian/rules b/debian/rules
index 615963d..d9d0fa4 100755
--- a/debian/rules
+++ b/debian/rules
@@ -764,33 +764,10 @@ minimal-test:
 		test_codecencodings_tw test_codecs test_multibytecodec \
 
 stamps/stamp-doc-html:
-	dh_testdir
-	$(MAKE) -C Doc html
-	@mkdir -p stamps
-	touch stamps/stamp-doc-html
+	# do nothing
 
 stamps/stamp-doc-info:
-	dh_testdir
-	rm -rf Doc/build/texinfo
-	$(MAKE) -C Doc texinfo
-# - Change name of info file (@setfilename) to include the version.
-# - Change the directory entry to include the version and point to the
-#   correct filename.
-# - Change image links (@image) to point to a subdirectory.
-# - Remove trailing whitespace.
-	sed -e 's/^@setfilename .*/@setfilename $(PVER).info/' \
-	    -e '/^@direntry/,/^@/s/^\*[^:]*: ([^)]*)/* Python $(VER): ($(PVER))/' \
-	    -e 's,@image{,@image{$(PVER)/,g' \
-	    -e 's/ *$$//' \
-	  < Doc/build/texinfo/python.texi \
-	  > Doc/build/texinfo/$(PVER).texi
-	rm -f Doc/build/texinfo/python.texi
-# - Create a symbolic link so that makeinfo will be able to find the
-#   image files.
-	ln -s . Doc/build/texinfo/$(PVER)
-	$(MAKE) -C Doc/build/texinfo
-	@mkdir -p stamps
-	touch stamps/stamp-doc-info
+	# do nothing
 
 build-doc: stamps/stamp-patch stamps/stamp-build-doc
 stamps/stamp-build-doc: stamps/stamp-doc-html stamps/stamp-doc-info
@@ -1406,42 +1383,7 @@ binary-indep: build-indep install stamps/stamp-control
 	dh_testdir -i
 	dh_testroot -i
 
-	: # $(p_doc) package
-	dh_installdirs -p$(p_doc) \
-		usr/share/doc/$(p_base) \
-		usr/share/doc/$(p_doc)
-	dh_installdocs -p$(p_doc)
-	cp -a Doc/build/html $(d_doc)/usr/share/doc/$(p_base)/
-	rm -f $(d_doc)/usr/share/doc/$(p_base)/html/_static/jquery.js
-	dh_link -p$(p_doc) \
-		/usr/share/doc/$(p_base)/html /usr/share/doc/$(p_doc)/html \
-		/usr/share/javascript/jquery/jquery.js /usr/share/doc/$(p_base)/html/_static/jquery.js \
-		/usr/share/javascript/underscore/underscore.js /usr/share/doc/$(p_base)/html/_static/underscore.js
-	dh_installinfo -p$(p_doc)
-	for f in $(d_doc)/usr/share/doc/$(p_base)/html/_images/*.png; do \
-	  b=$$(basename $$f); \
-	  dh_link -p$(p_doc) /usr/share/doc/$(p_base)/html/_images/$$b /usr/share/info/$(PVER)/$$b; \
-	done
-
-	: # devhelp docs
-	cd $(buildd_static) && ./python ../debian/pyhtml2devhelp.py \
-		../$(d_doc)/usr/share/doc/$(p_base)/html index.html $(VER) \
-		> ../$(d_doc)/usr/share/doc/$(p_base)/html/$(PVER).devhelp
-	gzip -9nv $(d_doc)/usr/share/doc/$(p_base)/html/$(PVER).devhelp
-	dh_link -p$(p_doc) \
-		/usr/share/doc/$(p_base)/html /usr/share/devhelp/books/$(PVER)
-
-	: # regenerate idlelib/help.html, see idlelib.help.copy_strip
-	sed 's/ *$$//' Doc/build/html/library/idle.html \
-		> $(d_idle)/usr/lib/python$(VER)/idlelib/help.html
-
-	for i in $(p_ltst); do \
-	  rm -rf debian/$$i/usr/share/doc/$$i; \
-	  ln -s $(p_base) debian/$$i/usr/share/doc/$$i; \
-	done
-
 	dh_installdebconf -i $(dh_args)
-	dh_installexamples -i $(dh_args)
 	-dh_icons -i $(dh_args) || dh_iconcache -i $(dh_args)
 	dh_installchangelogs -i $(dh_args)
 	dh_link -i $(dh_args)
@@ -1595,3 +1537,7 @@ binary: binary-indep binary-arch
 # Local Variables:
 # mode: makefile
 # end:
+
+override_dh_installdocs:
+	# do nothing
+
